frontend/awx/views/schedules/wizard/ScheduleEditWizard.tsx

```diff
@@ -0,0 +1,172 @@
+import { useTranslation } from 'react-i18next';
+import { useNavigate, useParams } from 'react-router-dom';
+import { PageHeader, PageLayout, PageWizard, PageWizardStep } from '../../../../../framework';
+import { useGetPageUrl } from '../../../../../framework/PageNavigation/useGetPageUrl';
+import { dateToInputDateTime } from '../../../../../framework/utils/dateTimeHelpers';
+import { AwxRoute } from '../../../main/AwxRoutes';
+import { ScheduleFormWizard } from '../types';
+import { awxErrorAdapter } from '../../../common/adapters/awxErrorAdapter';
+import { RulesStep } from './RulesStep';
+import { Frequency, RRule, RRuleSet, rrulestr } from 'rrule';
+import { ExceptionsStep } from './ExceptionsStep';
+import { SurveyStep } from '../../../common/SurveyStep';
+import { NodeTypeStep } from '../../../resources/templates/WorkflowVisualizer/wizard/NodeTypeStep';
+import { NodePromptsStep } from '../../../resources/templates/WorkflowVisualizer/wizard/NodePromptsStep';
+import { WizardFormValues } from '../../../resources/templates/WorkflowVisualizer/types';
+import { shouldHideOtherStep } from '../../../resources/templates/WorkflowVisualizer/wizard/helpers';
+import { RESOURCE_TYPE } from '../../../resources/templates/WorkflowVisualizer/constants';
+import { useGetItem } from '../../../../common/crud/useGet';
+import { Schedule } from '../../../interfaces/Schedule';
+import { awxAPI } from '../../../common/api/awx-utils';
+import { NodeReviewStep } from '../../../resources/templates/WorkflowVisualizer/wizard/NodeReviewStep';
+export function ScheduleEditWizard() {
+  const { t } = useTranslation();
+  const getPageUrl = useGetPageUrl();
+  const params = useParams<{ id?: string; schedule_id?: string }>();
+
+  const { data: schedule } = useGetItem<Schedule>(awxAPI`/schedules/`, params.schedule_id);
+
+  const navigate = useNavigate();
+
+  const [startDate, time]: string[] = dateToInputDateTime(
+    schedule?.dtstart as string,
+    schedule?.timezone
+  );
+
+  const handleSubmit = async (formValues: ScheduleFormWizard) => {
+    const { launch_config, prompt } = formValues;
+    const promptValues = prompt;
+    if (promptValues) {
+      if (launch_config) {
+        promptValues.original = {
+          launch_config,
+        };
+      }
+    }
+
+    await Promise.resolve();
+  };
+
+  const onCancel = () => navigate(-1);
```

Comment:
I'd suggest navigating to a specific location instead of calling `navigate(-1)`. If the navigation history href matches the current location, calling `navigate(-1)` will not navigate back. That is why the schedule wizard cancel button works on step 1 (history href is still the schedule list), but not on subsequent wizard steps (history href changes to current location).

Reply:
I updated this in [Fix cancel button navigate location](https://github.com/ansible/ansible-ui/pull/1932/commits/5af9146b5c021e4055ab9cbe506f4caa563e3a69)